de/manual/ch_Business.docbook

```diff
@@ -1990,616 +1990,1051 @@
+        Aufträge hilfreich sein.
+      </para>
+
+      <para>Um Lieferantenaufträge zu nutzen, müssen Sie diese über den Menüpunkt &mc.bus.ven.new-job;
+        erstellen. Es öffnet sich das Fenster <guilabel>Neuer Auftrag</guilabel> das die folgenden
+        Datenfelder enthält:
+      </para>
+
+      <figure pgwide="1">
+        <title>Einen neuen Lieferantenauftrag anlegen</title>
+
+        <screenshot>
+          <screeninfo>425x384</screeninfo>
+
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="figures/Busnss_Vendor_Job.png"
+                                   srccredit="Christian Wehling" />
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
 
-        <listitem>
-          <para><guilabel>Tax Table</guilabel> (optional) - this is a pop up menu of all the available tax tables.
-            If the item is taxable and tax has not been included in the unit price, then this tax
-            table is used to compute the amount of tax.
-          </para>
-        </listitem>
+      <glosslist>
+        <glossentry>
+          <glossterm>Auftragsinformationen - Auftragsnummer</glossterm>
+          <glossdef>
+            <para>(optional) - Geben Sie die Nummer für diesen Auftrag ein. Wenn Sie nichts eingeben, wird eine
+              Nummer entsprechend der Festlegung in den Buchoptionen
+              (<xref linkend="counters-book-options" />) für Sie ausgewählt.
+            </para>
+          </glossdef>
+        </glossentry>
 
-        <listitem>
-          <para><guilabel>Subtotal</guilabel> (uneditable) - computed subtotal for this item (less tax)
-          </para>
-        </listitem>
+        <glossentry>
+          <glossterm>Auftragsinformationen - Auftragsname</glossterm>
+          <glossdef>
+            <para>Der Name, den Sie dem neuen Auftrag zuweisen möchten.
+            </para>
+          </glossdef>
+        </glossentry>
 
-        <listitem>
-          <para><guilabel>Billable</guilabel> - is this item billable to the chargeback customer/job?
-          </para>
-        </listitem>
-      </itemizedlist>
+        <glossentry>
+          <glossterm>Mandanteninformationen - Lieferant</glossterm>
+          <glossdef>
+            <para>Der Lieferant, für den der Auftrag erstellt wird. Sobald Sie mit der Eingabe eines Namen beginnen,
+              zeigt &app; passende Einträge an. Andernfalls öffnet die Schaltfläche
+              <guibutton>Auswählen…</guibutton> den Dialog zur
+              <xref linkend="tool-find-bsnss" />, der Sie bei der Suche nach dem gewünschten
+              Lieferanten unterstützt.
+            </para>
+          </glossdef>
+        </glossentry>
 
-      <para>When you have finished entering all the items, <guilabel>Post</guilabel> the bill.
-      </para>
-    </sect2>
+        <glossentry>
+          <glossterm>Mandanteninformationen - Kurs</glossterm>
+          <glossdef>
+            <para>???
```

Comment:
Akzeptiert nur numerische Werte, und scheint bei Erreichen oder Überschreiten von 100 [%] den Auftrag als abgeschlossen zu markieren.

Derek fragen und in `Progress [%]` umbenennen. (Beide Dialoge)

Ein mögliches Beispiel wäre dafür: Als Zuckerbäcker brauche ich für eine Hochzeitstorte 100l Sahne und gehe auf den Markt, um von den Milchmänner solange ihre Sahne-Überschüße zu kaufen, bis ich genug habe.

Reply:
@derekatkins , @Bob-IT , can you explain the meaning of the "Rate" input field in the "New Job" dialog in the Business menu?

Is it supposed to be some kind of progress indicator?

Reply:
@CWehli It gets turned into a default unit price for bill/invoice line items.